main.py

```diff
@@ -11,7 +11,11 @@
 TOKEN = os.getenv("BOT_TOKEN")
 PREFIX = constants.COMMAND_PREFIX
 
-client = commands.Bot(commands.when_mentioned_or(PREFIX), case_insensitivity=True, owner_id=688275913535914014)
+SUPPORT_SERVER = "https://discord.gg/CgH6Sj6"
+INVITE = "https://discord.com/api/oauth2/authorize?client_id=715545167649570977&permissions=980675863&scope=bot"
```

Comment:
These things are defined in [**`constants.py`**](https://github.com/The-Codin-Hole/HotWired-Bot/blob/master/cogs/utils/constants.py) it would be wiser to use that instead of re-defining them here.

https://github.com/The-Codin-Hole/HotWired-Bot/blob/6e776c3b99088e27767ae2bafaa1f41d9bc93a23/cogs/utils/constants.py#L18-L28

Reply:
I haven't actually made that change. No idea why it shows up.